index.html

```diff
@@ -1,15 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <!-- HTML:
-    form - Select Input(Different Alcohols)
-    submit button - Give Me My Cocktail -->
-
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>What Cocktail Should I Make?</title>
+        <link rel="stylesheet" href="styles/styles.css" />
+    </head>
+    <body>
+        <main>
+            <header>
```

Comment:
Header element shouldn't go inside the main element - it should go before. 
(They have the same hierarchy in the page so shouldn't be nested)